src/RewindStreamException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Http\Client\Common\Plugin;
```

Comment:
Agreed. Putting the exception in `Http\Client\Common\Plugin\Exception\RewindStreamException` is preferred.